filter.js

```diff
@@ -13,12 +13,17 @@ const sentEventClick = (eventName) => {
     const eventList = await fetchData();
     // taka tablica jest niepotrzebna
     // zauważ że wtedy do każdego elementu dokładasz handler
+    const displayAmountOfEvent = (arrayList) => {
+      document.getElementById("eventNumber").innerHTML = arrayList.length;
```

Comment:
nitpick: taki handler - możesz wywalić w ogóle przed funkcję np.
`const eventNumberElement = document.getElementById("eventNumber")` 
a potem tylko 
`eventNumberElement.innerHTML = arrayList.length`

głównie optymalizuje to sięganie do DOMu za każdym razem :)